<ui version="4.0" >
 <class>PlotPreview</class>
 <widget class="QDialog" name="PlotPreview" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>437</width>
    <height>524</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" >
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" >
     <item>
      <widget class="QCheckBox" name="autoUpdate" >
       <property name="enabled" >
        <bool>false</bool>
       </property>
       <property name="text" >
        <string>Auto-update</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>28</width>
         <height>17</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="zoomIn" >
       <property name="enabled" >
        <bool>false</bool>
       </property>
       <property name="minimumSize" >
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="text" >
        <string/>
       </property>
       <property name="icon" >
        <iconset resource="icons.qrc" >
         <normaloff>:/icons/gtk-zoom-in.png</normaloff>:/icons/gtk-zoom-in.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="zoomOut" >
       <property name="enabled" >
        <bool>false</bool>
       </property>
       <property name="minimumSize" >
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="text" >
        <string/>
       </property>
       <property name="icon" >
        <iconset resource="icons.qrc" >
         <normaloff>:/icons/gtk-zoom-out.png</normaloff>:/icons/gtk-zoom-out.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="zoom1" >
       <property name="enabled" >
        <bool>false</bool>
       </property>
       <property name="minimumSize" >
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="text" >
        <string/>
       </property>
       <property name="icon" >
        <iconset resource="icons.qrc" >
         <normaloff>:/icons/gtk-zoom-100.png</normaloff>:/icons/gtk-zoom-100.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="zoomFit" >
       <property name="enabled" >
        <bool>false</bool>
       </property>
       <property name="minimumSize" >
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="text" >
        <string/>
       </property>
       <property name="icon" >
        <iconset resource="icons.qrc" >
         <normaloff>:/icons/gtk-zoom-fit.png</normaloff>:/icons/gtk-zoom-fit.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGraphicsView" name="imageView" >
     <property name="enabled" >
      <bool>false</bool>
     </property>
     <property name="renderHints" >
      <set>QPainter::Antialiasing|QPainter::TextAntialiasing</set>
     </property>
     <property name="resizeAnchor" >
      <enum>QGraphicsView::AnchorViewCenter</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2" >
     <item>
      <widget class="QComboBox" name="imageList" >
       <property name="enabled" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons" >
        <set>QDialogButtonBox::Close</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="icons.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>PlotPreview</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel" >
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>PlotPreview</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel" >
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
